libs/blocks/tabs/tabs.js

```diff
@@ -84,6 +87,77 @@ function initTabs(elm, config, rootElem) {
   if (config) configTabs(config, rootElem);
 }
 
+// Check if current tab is not visible and next tab is visible
+function previousTab(item, i, arr) {
+  const next = arr[i + 1];
+  return (!isTabInTabListView(item) && next && isTabInTabListView(next));
+}
+
+// Check if previous tab is visible and current tab is not visible
```

Comment:
Nit: this comment doesn't seem to apply to the `nextTab` function - though I know it's probably for the block of code below it.  I don't think it's needed.

Reply:
I re-wrote and simplified this comment, there was some confusion in the last PR and want to keep some explanation.